src/NuGet.Core/NuGet.Protocol/Resources/RegistrationResourceV3.cs

```diff
@@ -48,12 +48,12 @@ public RegistrationResourceV3(HttpSource client, Uri baseUrl)
         /// </summary>
         public virtual Uri GetUri(string packageId)
         {
-            if (String.IsNullOrEmpty(packageId))
+            if (string.IsNullOrEmpty(packageId))
```

Comment:
Revert file

Reply:
Fixes compile error.

Reply:
It should only be an IntelliSense error, if anything. Perhaps your VS is configured to treat as Error?
We don't typically modify files just for formatting, but I won't block on it.